eve/generate_junit_result.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# Simple ugly bash script to generate really really simple junit file
+# with just the result of the build step, waiting for something integrated
+# directly in `eve`
+
+FAILURE=0
+ERROR=0
+TYPE=""
+
+case "$FINAL_STATUS" in
+    "SUCCESSFUL")
+        ;;
+    "FAILED")
+        FAILURE=1
+        TYPE="failure"
+        ;;
+    *)
+        ERROR=1
+        TYPE="error"
+        ;;
+esac
+
+echo '<?xml version="1.0" ?>'
```

Comment:
Wouldn't it be much nicer to use a heredoc? :thinking:

Reply:
Yes true